test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -28,68 +28,53 @@ public static TheoryData TagStructureData
+                                AssemblyName = AssemblyName,
+                                TagStructure = TagStructure.WithoutEndTag
+                            },
+                            new TagHelperDescriptor
+                            {
+                                TagName = "p",
+                                TypeName = typeof(MultiSpecifiedTagStructureTagHelper).FullName,
+                                AssemblyName = AssemblyName,
+                                TagStructure = TagStructure.NormalOrSelfClosing
+                            }
                         }
                     },
                     {
                         typeof(MultiWithUnspecifiedTagStructureTagHelper),
                         new[]
                         {
-                            new TagHelperDescriptor(
-                                prefix: string.Empty,
-                                tagName: "input",
-                                typeName: typeof(MultiWithUnspecifiedTagStructureTagHelper).FullName,
-                                assemblyName: AssemblyName,
-                                attributes: Enumerable.Empty<TagHelperAttributeDescriptor>(),
-                                requiredAttributes: Enumerable.Empty<string>(),
-                                allowedChildren: null,
-                                tagStructure: TagStructure.WithoutEndTag,
-                                designTimeDescriptor: null),
-                            new TagHelperDescriptor(
-                                prefix: string.Empty,
-                                tagName: "p",
-                                typeName: typeof(MultiWithUnspecifiedTagStructureTagHelper).FullName,
-                                assemblyName: AssemblyName,
-                                attributes: Enumerable.Empty<TagHelperAttributeDescriptor>(),
-                                requiredAttributes: Enumerable.Empty<string>(),
-                                allowedChildren: null,
-                                tagStructure: TagStructure.Unspecified,
-                                designTimeDescriptor: null),
+                            new TagHelperDescriptor
+                            {
+                                TagName = "input",
+                                TypeName = typeof(MultiWithUnspecifiedTagStructureTagHelper).FullName,
+                                AssemblyName = AssemblyName,
+                                TagStructure = TagStructure.WithoutEndTag
+                            },
+                            new TagHelperDescriptor
+                            {
+                                TagName = "p",
+                                TypeName = typeof(MultiWithUnspecifiedTagStructureTagHelper).FullName,
+                                AssemblyName = AssemblyName,
+                                TagStructure = TagStructure.Unspecified
```

Comment:
No need to init `enum` property to its default value here either. Definitely no need when you're (unnecessarily) explicitly initializing the property in `TagHelperDescriptor` to the same value.